### PR TITLE
Confirmation page permutations

### PR DESCRIPTION
**Pull request by stevegraham**
Show different copy for different application states, e.g. remission,
normal, partial remission, payment failure, etc.

### CODE REVIEW

File: app/services/jadu/claim.rb [hunk 1]
```diff
@@ -43,8 +43,20 @@ def client
 
     def attachments
```
Comment (stevegraham) [inline app/services/jadu/claim.rb]: This issue is gnarly as f***. Confirmed on dev and prod, i.e. file and s3 backends.

**Comment by stevegraham** [comment]
@dan-palmer please review. :)

File: spec/rails_helper.rb [hunk 2]
```diff
@@ -42,6 +42,8 @@
   # https://relishapp.com/rspec/rspec-rails/docs
   config.infer_spec_type_from_file_location!
 
+  config.include ActiveSupport::Testing::TimeHelpers
```
Comment (dan-palmer) [inline spec/rails_helper.rb]: nice one :+1:

File: spec/factories.rb [hunk 3]
```diff
@@ -19,7 +19,7 @@
       Rack::Test::UploadedFile.new 'spec/support/files/file.csv'
     end
 
-    fee_group_reference { "%010d00" % rand(9999999999) }
+    fee_group_reference "511234567800"
```
Comment (dan-palmer) [inline spec/factories.rb]: :+1:

File: spec/features/confirmation_page_spec.rb [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+  before do
+    visit returning_user_session_path
+    fill_in_return_form claim.reference, 'lollolol'
+
+    visit claim_review_path
+    click_button 'Submit'
+  end
+
+  around do |example|
+    travel_to(Time.new 2014) { example.run }
```
Comment (dan-palmer) [inline spec/features/confirmation_page_spec.rb]: why not have the `travel_to` in the before block?

Comment (stevegraham) [inline spec/features/confirmation_page_spec.rb]: Don't have to reset it. I'm assuming this helper behaves like Timecop and needs resetting?

**Pull request merged**